README.md

```diff
@@ -1,57 +1,103 @@
-# Python Environments and Package Manager (experimental)
+# Python Environments (experimental)
 
 ## Overview
 
-The Python Environments and Package Manager extension for VS Code helps you manage Python environments and packages using your preferred environment manager backed by its extensible APIs. This extension provides unique support to specify environments for specific files or whole Python folders or projects, including multi-root & mono-repos scenarios.
+The Python Environments extension for VS Code helps you manage Python environments and packages using your preferred environment manager, backed by its extensible APIs. This extension provides unique support for specifying environments for specific files, entire Python folders, or projects, including multi-root and mono-repo scenarios. The core feature set includes: 
 
-> Note: This extension is in preview and its APIs and features are subject to change as the project continues to evolve.
+- 🌐 Create, delete, and manage environments
+- 📦 Install and uninstall packages within the selected environment
+- ✅ Create activated terminals
+- 🖌️ Add and create new Python projects
 
-> Important: This extension currently requires the pre-release version of the Python extension (ms-python.python) to operate (version 2024.23.2025010901 or later).
+> **Note:** This extension is in preview, and its APIs and features are subject to change as the project evolves.
+
+> **Important:** This extension requires the pre-release version of the Python extension (`ms-python.python`) to operate (version 2024.23.2025010901 or later).
 
 ## Features
 
-<img src=https://raw.githubusercontent.com/microsoft/vscode-python-environments/main/images/python-envs-overview.gif width=734 height=413>
+The "Python Projects" fold shows you all of the projects that are currently in your workspace and their selected environments. From this view you can add more files or folders as projects, select a new environment for your project, and manage your selected environments.
```

Comment:
We should elaborate that a project in this case is anything that requires its own python environment.

Reply:
I go into detail about a project starting on line 59, is this ok?